wordpress-mobile/WordPress-Android
# Feature/awesome accessibility font size audit

Fixes #7295 

Android is already pretty good at handling large font size and display elements, so we only had a couple of places where fixes were necessary.

- Stats legend.
- My Site blavatar and Site picker.
- EditText used in Sign Up/In flow.
- People list and details.
- Comments actions.
- Reader tag input field.

Hope I didn't miss anything important.

To test:
Go to accessibility settings on your device.
Change Font Size to "Largest"
Change Display Size to "Large"
Check out the app.
